### DISCUSSION TITLE
Progress Bar starts at 0:00 after 2 seconds

### DISCUSSION THREAD

**Discussion by mzurhorst** [music-assistant/support#2390] [Q&A] [answered]
Hi.

I am new to Music Assistant and puzzled, that I do not find a discussion for this obvious topic:  
For me, the progress bar is in a loop and progresses only for the first 2 seconds before it resets back to 00:00 again.

As far as I can observe, the album is still played, song after song, but Music Assistant does not display the next titlle, nor does it display all titles played.

![grafik](https://github.com/music-assistant/hass-music-assistant/assets/575190/2b319f25-5b22-467d-a92b-0fc54744936a)

**Comment by OzGav** [comment]
No one has discussed it because no one else is reporting it. You will need to provide more details particularly what player type.

**Reply by mzurhorst** [reply]
Oh, I did not expect this. 
So, I have toggled the UI to English that we are on the same page:

1)  I only have 1 player active.  Its type is "Home Assistant Media Players"
![grafik](https://github.com/music-assistant/support/assets/575190/eb638c5f-52ea-47f2-b2c1-5a9eb8a2e041)

2)  This player is a Linkplay-based player  (using the HACS custom integration).  Within Home Assistant, the progress bar behaves as expected.
![grafik](https://github.com/music-assistant/support/assets/575190/aede22a9-0b4e-4c39-b544-00b060c6579d)

**Reply by OzGav** [reply, marked as answer]
I draw your attention to this from the docs
![image](https://github.com/music-assistant/support/assets/19848947/b408ab5b-4292-478a-8203-d6c55058c577)

So linkplay is not currently natively supported but it is on the roadmap for a future release. 

You need to raise an issue report for the player behaviour and it will get looked at in due course.

**Reply by mzurhorst** [reply]
Ok, will do.  Thank you.

**Comment by mzurhorst** [comment]
https://github.com/music-assistant/hass-music-assistant/issues/2396
